.github/workflows/compare_storage_layout.py

```
@@ -1,68 +1,81 @@
+
+    # Compare non-gap variables
+    for index, row in old_layout.iterrows():
+        if row['Name'] != '__gap':
+            current_row = new_layout.loc[new_layout['Name'] == row['Name']]
+            if current_row.empty:
+                mismatches.append(f"Variable {row['Name']} is missing in the current layout")
+            elif not current_row.iloc[0].equals(row):
+                mismatches.append(f"Variable {row['Name']} has changed")
+    
+    if not mismatches:
+        print(f"No mismatches found")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)